convert2rhel/backup/__init__.py

```diff
@@ -70,6 +70,12 @@ def push(self, restorable):
         if not isinstance(restorable, RestorableChange):
             raise TypeError("`%s` is not a RestorableChange object" % restorable)
 
+        # check if the file is already added
```

Comment:
I think in this place we cannot expect only file, but there can be any other restorable. What if we would try to back up e.g. package - there isn't any `restorable.backup_path`.

Maybe it's a corner case, but I won't "lock" it just to the files.

Reply:
Ah right, yeah wording can probably be changed then. This is just checking if the two items are identical. By default two different class instances will be different unless `__eq__` is implemented like in RestorableFile

Reply:
Hmm, so maybe I would also add a comment saying that we have `__eq__` implemented only in the restorablefile, so working with the `backup_path` is safe